.github/workflows/build.yml

```diff
@@ -99,4 +101,5 @@ jobs:
           DD_API_KEY: ${{ secrets.DD_API_KEY }}
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          RUNTIME_PARAM: ${{ matrix.runtime-param }}
```

Comment:
Likewise here I think it would be nicer to just pass in a number like `10` -- it's always going to be node, so why type it every time? 🙂

Reply:
✅